dynatrace-oneagent-operator/README.md

```diff
@@ -0,0 +1,93 @@
+# Dynatrace OneAgent Operator Helm Chart
+
+This is the home of Dynatrace OneAgent Operator's Helm Chart which supports the rollout and lifecycle of [Dynatrace OneAgent](https://www.dynatrace.com/support/help/get-started/introduction/what-is-oneagent/) in Kubernetes and OpenShift clusters.
+Rolling out Dynatrace OneAgent via DaemonSet on a cluster is straightforward.
+Maintaining its lifecycle places a burden on the operational team.
+Dynatrace OneAgent Operator closes this gap by automating the repetitive steps involved in keeping Dynatrace OneAgent at its latest desired version.
+
+## Overview
+
+Dynatrace OneAgent Operator is based on [Operator SDK](https://github.com/operator-framework/operator-sdk) and uses its framework for interacting with Kubernetes and OpenShift environments.
+It watches custom resources `OneAgent` and monitors the desired state constantly.
+The rollout of Dynatrace OneAgent is managed by a DaemonSet initially.
+From here on Dynatrace OneAgent Operator controls the lifecycle and keeps track of new versions and triggers updates if required.
+
+![Overview](./overview.svg)
+
+
+## Supported platforms
+
+Depending of the version of the Dynatrace OneAgent Operator, it supports the following platforms:
+
+| Dynatrace OneAgent Operator Helm Chart version | Kubernetes | OpenShift Container Platform |
+| ---------------------------------------------- | ---------- | ---------------------------- |
+| v0.5.4                                         | 1.11+      | 3.11+                        |
+
+
+## Quick Start
+
+The Dynatrace OneAgent Operator acts on its separate namespace `dynatrace`.
+It holds the operator deployment and all dependent objects like permissions, custom resources and
+corresponding DaemonSets.
+To install the Dynatrace OneAgent Operator via Helm run the following command:
+
+### Adding Dynatrace OneAgent Helm repository
+```
+$ helm repo add dynatrace https://github.com/Dynatrace/helm-charts/repos/stable
+```
+
+
+#### Kubernetes
+```
+$ kubectl create namespace dynatrace
+$ helm install \
+    --name=dynatrace-oneagent-operator \
+    dynatrace/dynatrace-oneagent-operator \
+    -n dynatrace \
+    --set platform=kubernetes, \
```

Comment:
Just in case, the new lines here don't break the instruction, right?

Reply:
Splitting into multiple lines broke the command - changed it to single line and added quotes